internal/k8s-engine/controller/suite_test.go

```diff
@@ -1,4 +1,5 @@
-//+build controllertests
+//go:build controllertests
+// +build controllertests
```

Comment:
should I revert it? It's added because of go 1.17 which I use locally + in my IDE (which added that automatically) 

>The go command now understands //go:build lines and prefers them over // +build lines. The new syntax uses boolean expressions, just like Go, and should be less error-prone. As of this release, the new syntax is fully supported, and all Go files should be updated to have both forms with the same meaning. To aid in migration, gofmt now automatically synchronizes the two forms. For more details on the syntax and migration plan, see https://golang.org/design/draft-gobuild.



source: https://golang.org/doc/go1.17

Reply:
We can keep it or remove it. We need to migrate to Go 1.17 in the near future both locally and on CI anyway, so we can add the `//go:build` lines as a part of that small bump.

But, if you decide to keep it, then please also update all other files where we use build tags 👍